testutil/integration/simnet_test.go

```diff
@@ -492,7 +506,7 @@ func startTeku(t *testing.T, args simnetArgs, node int) simnetArgs {
 				args.ErrChan <- errors.New("beaconmock wasn't available after 10s")
 				return
 			default:
-				_, err := http.Get("http://" + args.VAPIAddrs[node])
+				_, err := http.Get("http://" + args.VAPIAddrs[node] + "/up")
```

Comment:
why add this?

Reply:
Without this, when it calls bmock to check if it up or not we get the following WARN logs from bmock:
![Screenshot 2023-05-29 at 12 21 23 PM](https://github.com/ObolNetwork/charon/assets/37813203/3ce74e12-a383-4205-8aab-35282fada917)

bmock support `/up` to check if bmock is up or not:
https://github.com/ObolNetwork/charon/blob/494f33fdf89f52618bb212c92e351a847f1ca6de/testutil/beaconmock/server.go#L62-L64